src/js/generalFunctions.js

```diff
@@ -0,0 +1,7 @@
+function generateDomElement(currentNode, element, ...classes) {
```

Comment:
...classes -> this is a bad idea to collect several class names. 
generateDomElement({...}. {...}, 1class, 2class, 3class, ..., 25class, 26class, ...)

I assume to collect classes to array and then send it as argument.
const classes = [1class, 2class, 3class, ..., 25class, 26class, ...];
generateDomElement({...}. {...}, classes);

Reply:
Done